metadata_backend/helpers/schemas/folders.json

```diff
@@ -22,6 +22,573 @@
       "type": "boolean",
       "title": "Published Folder"
     },
+    "doi": {
+      "type": "object",
+      "title": "The DOI schema",
+      "required": [
+        "identifier",
+        "url",
+        "creators",
+        "titles",
+        "publisher",
+        "subjects",
+        "publicationYear",
+        "resourceType"
+      ],
+      "properties": {
+        "identifier": {
+          "type": "object",
+          "title": "identifier object",
+          "required": [
+            "identifierType",
+            "doi"
+          ],
+          "properties": {
+            "identifierType": {
+              "type": "string",
+              "title": "Type of identifier (= DOI)"
+            },
+            "doi": {
+              "type": "string",
+              "title": "A persistent identifier for a resource"
+            }
+          }
+        },
+        "url": {
```

Comment:
we can keep this, but don't require it. We will add some configuration in the application how to handle this probably in a different issue, changing this should not be allowed

Reply:
part of `extraInfo`